### PR TITLE
Rename fluffy fixture

### PR DESCRIPTION
**Pull request by diitaz93**
## Description
Fluffy fixture `fastq_file_path` in `tests/cli/workflow/fluffy/conftest.py` shadows another fixture of the same name in `tests/meta/demultiplex/conftest.py`

### Changed

- Name of fixture `fastq_file_path` to `fluffy_fastq_file_path`

## Review

- [x] "Merge and deploy" approved by IO
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Deploy this branch on
  - [ ] stage
  - [ ] production

### CODE REVIEW

**Comment by diitaz93** [comment]
Nothing to test as it the change was on tests

File: tests/meta/demultiplex/test_utils.py [hunk 1]
```diff
@@ -330,27 +331,27 @@ def test_is_syncing_complete_false(
     assert not is_directory_synced
 
 
-def test_add_flow_cell_name_to_fastq_file_path(bcl2fastq_flow_cell_id: str, fastq_file_path: Path):
+def test_add_flow_cell_name_to_fastq_file_path(bcl2fastq_flow_cell_id: str, fluffy_fastq_file_path):
```
Comment (islean) [inline tests/meta/demultiplex/test_utils.py:334]: Are we sure that these tests used the renamed Path? Unless it is directly imported, I don't think this file can access the fixture.

**Review by islean — COMMENTED** [review]
I like the initiative but I don't know if the tests wherein the usage was changed actually used the changed fixture.

File: tests/meta/demultiplex/test_utils.py (hunk 1 above)
Comment (diitaz93) [reply tests/meta/demultiplex/test_utils.py]: fixed

**Review by islean — APPROVED** [review]
🔥

**Comment by diitaz93** [comment]
## Deployed to stage:
```shell
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/js.diazboada
done.
Log deploy... done.
cg, version 49.0.10
θ67° 1d [js.diazboada@hasta:~] [S_base] 4m8s $
```
## Deployed to production:
```shell
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/js.diazboada
done.
Log deploy... done.
cg, version 49.0.10
```

**Pull request merged**